Frontend/CO.CDP.OrganisationApp/Constants/ApiExceptionMapper.cs

```diff
@@ -0,0 +1,37 @@
+using CO.CDP.Organisation.WebApiClient;
+using Microsoft.AspNetCore.Mvc.ModelBinding;
+
+namespace CO.CDP.OrganisationApp.Constants;
+
+public static class ApiExceptionMapper
+{
+    public static void MapApiExceptions(
+        ApiException<ProblemDetails> aex,
+        ModelStateDictionary modelState)
+    {
+        var code = ExtractErrorCode(aex);
+
+        if (!string.IsNullOrEmpty(code))
```

Comment:
What will happen if code is null or empty? I'm guessing no error will be shown. Since this method is called when an exception is thrown, is it what we expect?

Reply:
refactored the code